core/src/main/java/io/seata/core/constants/ConfigurationKeys.java

```diff
@@ -92,4 +92,81 @@
      * The constant CLIENT_REPORT_RETRY_COUNT.
      */
     public static final String CLIENT_REPORT_RETRY_COUNT = CLIENT_PREFIX + "report.retry.count";
+
+    /**
+     * The constant STORE_DB_GLOBAL_TABLE.
+     */
+    public static final String STORE_DB_GLOBAL_TABLE  = "store.db.global.table";
+
+    /**
+     * The constant STORE_DB_BRANCH_TABLE.
+     */
+    public static final String STORE_DB_BRANCH_TABLE  = "store.db.branch.table";
+
+    /**
+     * The constant STORE_DB_GLOBAL_DEFAULT_TABLE.
+     */
+    public static final String STORE_DB_GLOBAL_DEFAULT_TABLE  = "global_table";
+
+    /**
+     * The constant STORE_DB_BRANCH_DEFAULT_TABLE.
+     */
+    public static final String STORE_DB_BRANCH_DEFAULT_TABLE  = "branch_table";
+
+    /**
+     * The constant STORE_DB_DATASOURCE_TYPE.
+     */
+    public static final String STORE_DB_DATASOURCE_TYPE  = "store.db.datasource-type";
+
+
+    /**
+     * The constant STORE_DB_TYPE.
+     */
+    public static final String STORE_DB_TYPE  = "store.db.db-type";
```

Comment:
what  is  the difference of datasource-type and db-type? maybe it is confused

Reply:
db-type is  mysql/oracle/oceanbase etc.
datasource-type is the implement of javax.sql.DataSource , such as DruidDataSource/DbcpDataSource etc.

Reply:
I had changed  `datasource-type ` to `datasource`